tests/unit/fem/test_get_value.cpp

```diff
@@ -287,7 +287,7 @@ TEST_CASE("2D GetValue",
 {
    int log = 1;
    int n = 1;
-   int dim = 2;
+   constexpr int dim = 2;
```

Comment:
I made some changes to this unit test in PR #1529 which should make this change unnecessary.  If this PR makes it through review sooner I can remove the `constexpr` in PR #1529 later.

Reply:
> Would `const` work just as well or does this newer declaration make a difference?

`const` should be okay here as well since the value is known at compile time.